### PR TITLE
add null type to PopperProps

### PR DESCRIPTION
**Pull request by smonero**
### Description

This is useful to be more like the material-ui popper

### GitHub Issue

Fixes #

### CODE REVIEW

**Review by scarlettperry — APPROVED** [review]

File: frontend/packages/core/src/popper.tsx [hunk 1]
```diff
@@ -89,7 +89,7 @@ export interface PopperProps
   extends Pick<ClickAwayListenerProps, "onClickAway">,
     Pick<MuiPopperProps, "placement"> {
   open: boolean;
-  anchorRef: React.MutableRefObject<HTMLElement>;
+  anchorRef: React.MutableRefObject<HTMLElement> | null;
```
Comment (danielhochman) [inline frontend/packages/core/src/popper.tsx:92]:
```suggestion
  anchorRef?: React.MutableRefObject<HTMLElement>;
```

Comment (smonero) [reply frontend/packages/core/src/popper.tsx:92]: https://stackoverflow.com/questions/57243477/can-an-optional-parameter-be-null-in-typescript

Item 5 shows that this approach of using optional to allow for null does not work (it does for `undefined` but not `null`)

Comment (danielhochman) [reply frontend/packages/core/src/popper.tsx:92]: ah true, can we have both then? i think both values make sense i.e. `anchorRef?: React.MutableRefObject<HTMLElement> | null`

**Review by danielhochman — APPROVED** [review]

File: frontend/packages/core/src/popper.tsx (hunk 1 above)
Comment (smonero) [reply frontend/packages/core/src/popper.tsx:92]: we can

**Pull request merged**